build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id "com.github.johnrengelman.shadow" version "6.1.0" apply false
+}
+
+subprojects {
+    apply plugin: "java"
+    apply plugin: "maven-publish"
+
+    sourceCompatibility = 11
+    targetCompatibility = 11
+    version rootProject.file(".loomversion").text
+
+    ext {
+        mcVersion = version
+        jbsdiffVersion = "5df6d1a09b"
```

Comment:
This should probably be moved to the launcher specific build file, since it isn't used anywhere else.

Reply:
It doesn't work otherwise, and I can't work out why.